cfe_internal/enterprise/ha/ha_info.json

```diff
@@ -1,15 +1,15 @@
 {
  "192.168.100.10":
  {
-  "sha": "3e9df4ec8c9826eaa2c397f99eeae22e0a35a22307474d56f7fdedadff7dd485",
+  "sha": "PLACE KEY HERE",
   "internal_ip": "192.168.100.10",
-  "tags": ["node1"]
+  "is_in_cluster" : true,
  },
  "192.168.100.11":
  {
-  "sha": "37baa6f2f18a907131dadbf3e255d4ed0eda2f403e2123bab2c5f82f7ac8cd79",
+  "sha": "PLACE KEY HERE",
   "internal_ip": "192.168.100.11",
-  "tags": ["node2"]
+  "is_in_cluster" : true,
  }
```

Comment:
Do you think this should be example_ha_info.json or something like what we did with def.json in the root of the repository?